src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -274,4 +279,24 @@ const AnimatedWrap = styled(animated.div)`
   ${breakpoint('medium', `padding-bottom:0;`)}
 `
 
-export default React.memo(AnimatedGlobalPreferences)
+const GlobalPreferencesWithDependencies = React.memo(
+  function GlobalPreferencesWithDependencies(props) {
+    const { optedOut } = useHelpScout()
+    const { layoutName } = useLayout()
+    return (
+      <ToastHub
+        shift={optedOut ? 0 : layoutName === 'small' ? 5.5 * GU : 6.5 * GU}
```

Comment:
This is kind of weird, because the helpscout beacon is always there (also when apps are loaded) and it's only the preferences that are getting this shift. Maybe changing the default in aragonUI would be better cc @dizzypaty?

Reply:
If the user opts out of the Help Scout beacon then there is no need for the shift.

Reply:
Good point, which means having to communicate this information to apps :-(

Maybe we could expose a toast service exposed to apps, that they could use instead of `Toast`? That way the client will always know how to position it.

Reply:
@sohkai Yes, this should definitely be a global behavior, not only apply when toasts are display on Global pref.

Reply:
> Maybe we could expose a toast service exposed to apps, that they could use instead of Toast? That way the client will always know how to position it.

Maybe Toast could figure out if it is inside client and then trigger the call to the client to show the toast instead of showing it locally.

Reply:
Yes totally! Once we’ll start having components aware of aragonAPI, we could do it transparently.

Reply:
Maybe... we should just move the HelpScout beacon to somewhere else that won't collide with a toast? Having apps communicate their toast positioning sounds really unnecessary.

In any case, I originally had the impression toasts would become an aragonAPI-level API, rather than each app using the component directly, so perhaps migrating completely to that would alleviate things.